python/dcr_template.py

```diff
@@ -25,21 +25,24 @@
 This is a work in progress!
 """
 from __future__ import print_function, division, absolute_import
+from functools import reduce
 import imp
 import numpy as np
 from scipy import constants
 from scipy.ndimage.interpolation import shift as scipy_shift
 import scipy.optimize.nnls as positive_lstsq
 
 import lsst.daf.persistence as daf_persistence
-import lsst.afw.geom as afwGeom
+# import lsst.afw.geom as afwGeom
 import lsst.afw.image as afwImage
+import lsst.afw.math as afwMath
+import lsst.meas.algorithms as measAlg
 import lsst.pex.policy as pexPolicy
 # from lsst.pipe.base.struct import Struct
 from lsst.sims.photUtils import Bandpass, PhotometricParameters
 from lsst.utils import getPackageDir
 imp.load_source('calc_refractive_index', '/Users/sullivan/LSST/code/StarFast/calc_refractive_index.py')
```

Comment:
I guess this is just test code that runs only on your machine?
to make it more portable you might consider
```
from os.path import expander
home = expanduser("~")
```
or even better, set starfast_simulator up as an eups package and use getPackageDir to find it, and a relative path from there to calc_refractive_index.py.
Whether to do all that, of course depends on how much work it's worth putting into your development & test environment of course. But it would have made it a little easier to share with me last week if it was set up a little more formally. It'll also be easier for you if you every e.g. switch development machines.